performance_regression_test.py

```diff
@@ -42,9 +47,9 @@ def display_single_result(self, result):
                                           result['Total partitions'],
                                           result['Total errors']))
 
-    def get_test_xml(self, result):
+    def get_test_xml(self, result, test_name='simple_regression_test'):
```

Comment:
@abvgedeika Thanks for your review, I already merged the PR #249. Can you rebase your code to latest master, and update your branch `ES_json1`?